docs/src/lib/autodiff.md

```diff
@@ -3,16 +3,30 @@ CurrentModule = ExaPF.AutoDiff
 ```
 # AutoDiff
 
-## Abstract Types
+## Variables
+```@docs
+AbstractStack
+
+```
+
+## Expressions
+
+```@docs
+AbstractExpression
+adjoint!
+
+```
+
+
+## First and second-order derivatives
 
 ```@docs
+AbstractJacobian
 AbstractHessianProd
 AbstractFullHessian
-AbstractJacobian
-AbstractStack
```

Comment:
No more stack?

Reply:
The stack has been moved right before in the file `autodiff.md` :)